src/DashboardService.Models.Db/DbBoard.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.DashboardService.Models.Db;
+
+public class DbBoard
+{
+  public const string TableName = "Boards";
+
+  public Guid Id { get; set; }
+  public Guid ProjectId { get; set; }
+  public string Name { get; set; }
+  public bool IsActive { get; set; }
+  public Guid CreatedBy { get; set; }
+  public DateTime CreatedAtUtc { set; get; }
+  public Guid? ModifiedBy { get; set; }
+  public DateTime? ModifiedAtUtc { get; set; }
+
+  public ICollection<DbGroup> Groups { get; set; }
```

Comment:
assign new List() to avoid null reference

Reply:
++